pyproject.toml

```diff
@@ -26,6 +26,8 @@ archs = ["x86_64", "arm64"]
 addopts = "-ra -v"
 filterwarnings = [
   'ignore:You are running a "Debug" build of scipp:',
+  'ignore:sc.matrix\(\) has been deprecated:DeprecationWarning',
+  'ignore:sc.matrices\(\) has been deprecated:DeprecationWarning',
```

Comment:
Can we fix these errors instead of ignoring them?

Reply:
Not until we actually remove `matrix` and `matrices`. Unless you want to remove tests for them.